docs/UserGuide.md

```diff
@@ -305,8 +305,8 @@ Example:
 
 > :exclamation: Notes about the recipe files
 > * A recipe will correspond to a recipe file which is a plaintext file with the name `<recipeName>.txt` where `recipeName` is the name of the recipe.
-> * It is highly recommended that you only modify recipe files through the `recipe` commands in `Decodex` and not manually edit the recipe files.
-> * If you choose to manually edit the recipe files, they should not be edited while `Decodex` is running as `Decodex` could accidentally overwrite your changes.
+> * It is highly recommended that you only modify recipe files through the [`recipe`](#recipe-commands-recipe) commands in Decodex and not manually edit the recipe files.
+> * If you choose to manually edit the recipe files, the correct format would be that each line in the respective recipe file should contain only one module, and they should be formatted in the following syntax of `<moduleName> {moduleArgument}`. You should also not be editing while Decodex is running as Decodex could accidentally overwrite your changes.
 > * Take note that incorrectly edited files will be considered as invalid and will not be loaded on program start up.
 > * Additional note would be that the recipe files would not be loaded successfully if Decodex does not have access rights to the files, or it is not in the valid filetype.
```

Comment:
Since these points are already considered notes, probably remove the parts "Take note" or "additional note".